OJS.Workers.SubmissionProcessors/ExecutionTypeFilters/SubmissionFilteringServiceBase.cs

```diff
@@ -23,11 +23,11 @@ protected SubmissionFilteringServiceBase()
 
         protected abstract ISet<CompilerType> DisabledExecuteAndCompileCompilerTypes { get; }
 
-        public bool CanProcessSubmission(IOjsSubmission submission, ISubmissionWorker submissionWorker)
+        public WorkerStateForSubmission GetWorkerStateForSubmission(IOjsSubmission submission, ISubmissionWorker submissionWorker)
         {
             if (submission == null)
             {
-                return false;
+                return WorkerStateForSubmission.NullableSubmission;
```

Comment:
Submission cannot/should not be null at this point, so you can remove this check.

Reply:
Done